tests/integration/long/test_failure_types.py

```diff
@@ -71,7 +71,7 @@ def setUp(self):
                 "Native protocol 4,0+ is required for custom payloads, currently using %r"
                 % (PROTOCOL_VERSION,))
         try:
-            self.cluster = Cluster(protocol_version=ProtocolVersion.MAX_SUPPORTED, allow_beta_protocol_version=True)
+            self.cluster = Cluster(protocol_version=ProtocolVersion.MAX_SUPPORTED)
```

Comment:
I'd expect this to fail still since now we will try to instantiate Cluster with `ProtocolVersion.V5` (because `ProtocolVersion.MAX_SUPPORTED = max(V5, V4, V3, V2, V1)`) but without setting `allow_beta_protocol_version=True`, I think we have to remove the whole `try ... except` and just do:
`self.cluster = Cluster(protocol_version=PROTOCOL_VERSION)`

Reply:
The try/except logic makes the test pass, but you're right that there's no point to this now. I'll remove the try/except.